pkgs/development/python-modules/linien-client/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, linien-common
+, setuptools
+, fabric
+, numpy
+, scipy
+}:
+
+buildPythonPackage rec {
+  pname = "linien-client";
+  pyproject = true;
+
+  inherit (linien-common) src version;
+
+  sourceRoot = "source/linien-client";
+
+  nativeBuildInputs = [ setuptools ];
+
+  propagatedBuildInputs = [
+    fabric
+    numpy
+    scipy
+    linien-common
+  ];
+
```

Comment:
Whats with tests and the pythonImportsCheck?

Reply:
`pythonImportsCheck` added